tracer/test/Datadog.Trace.IntegrationTests/OriginTagSendTracesCollection.cs

```diff
@@ -0,0 +1,19 @@
+// <copyright file="OriginTagSendTracesCollection.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using Datadog.Trace.Agent;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.TestHelpers;
+using Xunit;
+
+namespace Datadog.Trace.IntegrationTests
+{
+    [CollectionDefinition(nameof(OriginTagSendTracesCollection), DisableParallelization = true)]
+    public class OriginTagSendTracesCollection
```

Comment:
I don't see this being used anywhere, was this supposed to be used somewhere like: `[Collection(nameof(OriginTagSendTracesCollection))]`

Reply:
Right I thought I would be need it. but nope, removing it.

Reply:
done in [6a0e661](https://github.com/DataDog/dd-trace-dotnet/pull/2402/commits/6a0e661a6fc2e2d4350db5823d5fb7d5cb09e5e8)